services/datamanager/builtin/builtin.go

```diff
@@ -62,19 +62,23 @@ const defaultCaptureQueueSize = 250
 // Default bufio.Writer buffer size in bytes.
 const defaultCaptureBufferSize = 4096
 
+// Default time to wait in milliseconds to check if a file has been modified.
+const defaultFileLastModifiedMillis = 10000.0
+
 var clock = clk.New()
 
 var errCaptureDirectoryConfigurationDisabled = errors.New("changing the capture directory is prohibited in this environment")
 
 // Config describes how to configure the service.
 type Config struct {
-	CaptureDir            string                           `json:"capture_dir"`
-	AdditionalSyncPaths   []string                         `json:"additional_sync_paths"`
-	SyncIntervalMins      float64                          `json:"sync_interval_mins"`
-	CaptureDisabled       bool                             `json:"capture_disabled"`
-	ScheduledSyncDisabled bool                             `json:"sync_disabled"`
-	Tags                  []string                         `json:"tags"`
-	ResourceConfigs       []*datamanager.DataCaptureConfig `json:"resource_configs"`
+	CaptureDir             string                           `json:"capture_dir"`
+	AdditionalSyncPaths    []string                         `json:"additional_sync_paths"`
+	SyncIntervalMins       float64                          `json:"sync_interval_mins"`
+	CaptureDisabled        bool                             `json:"capture_disabled"`
+	ScheduledSyncDisabled  bool                             `json:"sync_disabled"`
+	Tags                   []string                         `json:"tags"`
+	ResourceConfigs        []*datamanager.DataCaptureConfig `json:"resource_configs"`
+	FileLastModifiedMillis int                              `json:"file_last_modified_seconds"`
```

Comment:
I think the json tag should say millis instead of seconds

Reply:
Good catch!